bessctl/commands.py

```diff
@@ -694,11 +694,18 @@ def _do_start(cli, opts):
     except subprocess.CalledProcessError:
         try:
             cli.bess.connect()  # reconnect to the old instance, if any
-        except:
+        except cli.bess.APIError:
```

Comment:
Can RPCError occur here?

It might be nice to have a distinguished "connection failed or rejected" error as APIError is also used for bad arguments, but since we only wait 3 seconds that's not critical.

Reply:
`RPCError` cannot occur here, but maybe `APIError` should have been `RPCError` as you pointed out.